config.go

```diff
@@ -66,6 +66,7 @@ type Settings struct {
 	AWSInstanceRole     string                  `yaml:"aws_instance_role" bson:"aws_instance_role" json:"aws_instance_role"`
 	Banner              string                  `bson:"banner" json:"banner" yaml:"banner"`
 	BannerTheme         BannerTheme             `bson:"banner_theme" json:"banner_theme" yaml:"banner_theme"`
+	Bucket              BucketConfig            `bson:"bucket" json:"bucket" yaml:"bucket" id:"bucket"`
```

Comment:
I would think to call it something like LoggingBucket or something to make it plain what the bucket is for.

Reply:
The idea is that we can set up all of our buckets with this single config (say, in the future, we may want to add the test results bucket here).

Reply:
Ah, I understand.
Maybe `Buckets`, then?